packages/salesforce/test/Utils.test.js

```diff
@@ -0,0 +1,108 @@
+import { expect } from 'chai';
+import { flattenData, validateNoDotKeys } from '../src/Utils';
+
+describe('flattenData', () => {
```

Comment:
Great tests, thank you!

I was a bit unsure about what was going on here - but I was able to look at these tests and they explained things beautifully.